ee/maintained-apps/outputs/intune-company-portal/darwin.json

```diff
@@ -1,21 +1,21 @@
 {
   "versions": [
     {
-      "version": "5.2508.0",
+      "version": "5.2508.1",
       "queries": {
         "exists": "SELECT 1 FROM apps WHERE bundle_identifier = 'com.microsoft.CompanyPortalMac';"
       },
-      "installer_url": "https://officecdn.microsoft.com/pr/C1297A47-86C4-4C1F-97FA-950631F94777/MacAutoupdate/CompanyPortal_5.2508.0-Upgrade.pkg",
-      "install_script_ref": "79b2c92d",
+      "installer_url": "https://officecdn.microsoft.com/pr/C1297A47-86C4-4C1F-97FA-950631F94777/MacAutoupdate/CompanyPortal_5.2508.1-Upgrade.pkg",
+      "install_script_ref": "a407504a",
       "uninstall_script_ref": "773a6033",
-      "sha256": "67dac5cffc5620ee6a97ff8d445a1880f8c2a2c9488b7e322a0dc453fbf27842",
+      "sha256": "1122aca4be1798d8a21bfab165419a74ed80af9d28b4eed459cdc6e6c224d7a3",
```

Comment:
This sha256 does not match the installer. See https://github.com/fleetdm/fleet/issues/33896.